EntAppProject/src/main/java/com/jobook/EntAppProject/Services/IJobService.java

```diff
@@ -8,15 +8,17 @@
 public interface IJobService {
 	/**
 	 * Get all jobs that related to userId provided by looking at the employee and customer Id
-	 * @param userId
-	 * @return
+	 * @param userId the user id to get all the jobs for
+	 * @return a list of jobs that the user has
+	 * @throws Exception any exception in the underlying persistence
 	 */
-	List<Job> getUsersRelatedJobs(int userId);
-	
+	List<Job> getUsersRelatedJobs(int userId) throws Exception;
+
 	/**
 	 * Searches all jobs by a search term and returns jobs that the short description contains the search term
-	 * @param searchTerm
-	 * @return
+	 * @param searchTerm the term to find the job by
+	 * @throws Exception any exception in the underlying persistence
+	 * @return list of jobs that matched the searchTerm
 	 */
-	List<Job> searchJobByName(String searchTerm);
+	List<Job> searchJobByName(String searchTerm) throws Exception;

```

Comment:
These methods should throw exceptions? So make sure you put that in the signature.

Reply:
This comment applies to a lot of other lines

Reply:
Does the implementing class throw an Exception?  If so, then yes, add this.  If not, then no.